### ISSUE TITLE
Published student pricing, packages, and certifications

### ISSUE THREAD

**Issue by blizzardengle** [JackLot/LearnGit.io#35]
**Background**

I teach at a university, and we are constantly looking for outside resources that would be a value add to our students and the department. Teaching version control to every one of our students is an area we are looking to improve in. Our goal is that every student who graduates from our program has a certain level of guaranteed Git mastery. LearnGit.io seems to have the potential to help us with this goal.

**Request**

To help mine and other universities adopt something like this, I would like to propose:

1. Publicly posted discounted student prices.
2. Publicly posted discounted teacher/department prices.
3. Splitting access into packages to further reduce costs for students:
    - For example, lessons 1 - 15 could be what my department would be most interested in.
    - Lessons 16 and beyond would be an optional or only for students in certain advanced courses.
4. UPDATE: A voucher system where we can purchase access for a student. This would be needed when a student can not afford the material and we purchase it on their behalf.
5. The university can easily offer a certificate for completing these, but if LearnGit.io had their own to offer, that would be a bonus.

**Comment by JackLot** [comment]
These are fantastic suggestions - thank you for taking the time to write them down. I really do think the visual learning style of LearnGit could be great for students, and I'd love to make it available more widely. 

I can implement 1 & 2 within the next week or so. Typically I do between 50% and 75% discount for students, but open to suggestions. I also think 4 is doable with my billing system so let me look into that. 

3 and 4 are on the roadmap, but will take a bit longer, but I'm willing to just make the all-in cost low enough for students in the meantime.

I'm on my way to Chicago for a long weekend, but would it be ok for me to email you next week when I get back? I have some other questions, and I think you'd have good advice.

Cheers,
Jack

**Comment by blizzardengle** [comment]
> would it be ok for me to email you next week when I get back?

Absolutely! I look forward to connecting with you to talk about this more. You can send me an [email](mailto:keersc@byui.edu) or [schedule a meeting](https://calendly.com/chris-keers/meet) with me if you like.

Enjoy your weekend!

**Comment by JackLot** [comment]
Thanks! I scheduled a chat for next ~~Thursday~~ Wednesday.

**Comment by JackLot** [comment]
Using this issue to track progress on some of the suggestions listed above. Publicly posted student discounts are now live on the homepage.

![Screenshot 2024-06-07 at 3 11 03 PM](https://github.com/JackLot/LearnGit.io/assets/1841253/7024bf6f-a336-481e-8e81-1b52d445fb53)

**Comment by JackLot** [comment]
Applying for a student discount is now self-service via https://learngit.io/education/student-application
